nvflare/tool/job/job_cli.py

```diff
@@ -551,15 +543,18 @@ def prepare_meta_config(cmd_args):
             dst_path = meta_path
             break
 
+    src_meta_path = os.path.join(target_template_dir, "meta.conf")
+    print(src_meta_path)
+    if not os.path.isfile(src_meta_path):
+        dst_config = load_default_config_template("meta.conf")
+    else:
+        dst_config = CF.parse_file(src_meta_path)
+
     # Use existing meta.conf if user already defined it.
-    if not dst_path:
-        dst_config = load_src_config_template("meta.conf")
+    if not dst_path or (dst_path and cmd_args.force):
         dst_config.put("name", app_name)
         dst_path = os.path.join(job_folder, "meta.conf")
-    else:
-        dst_config = CF.from_dict(ConfigFactory.load_config(dst_path).to_dict())
-
-    save_config(dst_config, dst_path)
+        save_config(dst_config, dst_path)
```

Comment:
If source_meta_file exist, we replace something inside and write it out.
If source_meta_file does NOT exist, we init from default and write it out.

Why do we need to check `if not dst_path or (dst_path and cmd_args.force):`?